includes/callbacks.inc

```diff
@@ -60,6 +60,7 @@ function islandora_entities_query_entities_callback($type) {
  *   array of all candidates
  */
 function islandora_entities_autocomplete($type) {
+  module_load_include('inc', 'islandora_solr', 'includes/query_processor');
```

Comment:
Just as a note: includes/query_processor.inc is included in the files[] declarations in islandora_solr.info, so it'd be okay to cut out these module_load_include()s